### DISCUSSION TITLE
[Bug]: Issue with some natives.

### DISCUSSION THREAD

**Discussion by SuriMax2** [YimMenu/YimMenu#3174] [General]
### Describe the bug

Hi, i like the NativeUI, i have convert YimMenu to stop using ImGui, on theoryc i just removed the call to renders, and create a loop, that tick with the script_mgr, that create the menu, but having some issues, for example:

Here are some natives that i log that success current im just have the basic,  Title, Option, without textures, now that i triying to use a texture for create the Header with a default sprite from the game and give sound to the scroll, some natives like: 

GRAPHICS::REQUEST_STREAMED_TEXTURE_DICT
or AUDIO::PLAY_SOUND_FRONTEND crash under end_call

Others natives work correct like:
4594, 727, 727, 4531, 3343, 717, 3888 etc..

A example of the crash dump from the REQUEST_STREAMED_TEXTURE_DICT:
1.- HAS_STREAMED_TEXTURE_DICT_LOADED
2.- HAS_STREAMED_TEXTURE_DICT_LOADED  call  -> invoke- crash under:end_call
        `native_invoker::invoke<1064,0,void,char const * __ptr64 & __ptr64,int & __ptr64>`
        `native_invoker::end_call<1064,0>`
3.-  H:\Grand Theft Auto V\GTA5.exe+0xA8726A 0x7FF6042D726A
In case of this native i getting others dumps places like (3)

For the other native the same:
`
native_invoker::end_call<92,0>
native_invoker::invoke<92,0,void,int & __ptr64,char const * __ptr64 & __ptr64,char const * __ptr64 & __ptr64,int & __ptr64>
`

Others ref on dump after the end_call go called: 
stack dump [0]	 H:\Grand Theft Auto V\GTA5.exe+0x34230E 0x7FF603B9230E
stack dump [1]	 H:\Grand Theft Auto V\GTA5.exe+0x380BB4 0x7FF603BD0BB4
stack dump [2]	 H:\Grand Theft Auto V\GTA5.exe+0x381880 0x7FF603BD1880
stack dump [3]	 H:\Grand Theft Auto V\GTA5.exe+0xA700ED 0x7FF6042C00ED

### Steps To Reproduce

1.- Call native_invoker::end_call<1064,0> | AUDIO::PLAY_SOUND_FRONTEND

### Expected Behavior

Crash as soon this native and others go called.

### Logs

`

`

### Store

Rockstar

### Language

English

### YimMenu Language

English (US)

### Additional context

Edited version (i just cut the render instance).
Create a loop, and add to the script_mgr.
Nothing more different from the last yim build.

Just that.

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

### Coload

- [X] I've tested this without co-loading anything.

**Comment by xiaoxiao921** [comment]
Issues are only related to YimMenu unmodified, moving to discussion

**Reply by SuriMax2** [reply]
Sorry for that, my bad.
Any case i just download the source this morning, build and the plan was to move all the menu to NativeUI style cause is more easy for me, to navigate.

As say, i just stop the render call to stop the ImGui getting called.
Then i start by adding a new script to the script_mgr, that loops.

There i can use natives without issues, PLAYER_ID, and others but some of the natives with the invoker, end on a crash.
More far from that i dont modified any more to the lasted, some natives fail and crash ex:
PLAY_SOUND_FRONTEND |  GRAPHICS::REQUEST_STREAMED_TEXTURE_DICT.

**Comment by xiaoxiao921** [comment]
Can you revert the changes from this commit on your machine to see if that fixes it?

https://github.com/YimMenu/YimMenu/commit/fb07065aaf3d0777d4cacef3e211d3b3cd6a09db

**Reply by SuriMax2** [reply]
I have done the same on Horse Menu, for me cause i have a some fingers issues , i doe custom keys and native ui near my left hand to click and move on the nativeUI, basic i cant click soo fast or move the mouse soo fast, thats why i dont use the ImGui.

Gonna revert, and just declare PLAY_SOUND on some of the current Yim Loops

**Comment by SuriMax2** [comment]
The change to the old class work this issue, i have check the classes, partial i just cut alot of YimMenu, and make a simple UI version to used with my fingers, as i can. Noo to big, take me a few houts but is good
